WARNING - utils.py - 2021-09-30 12:12:35,764 - No checkpoint found at ./checkpoints-meta/checkpoint.pth. Returned the same state as input
INFO - run_lib.py - 2021-09-30 12:12:43,802 - Starting training loop at step 0.
INFO - run_lib.py - 2021-09-30 12:12:51,388 - step: 0, training_loss: 9.85061e+04
INFO - run_lib.py - 2021-09-30 12:12:52,356 - step: 0, eval_loss: 9.84156e+04
